README.md

```diff
@@ -191,6 +191,9 @@ kind: Service
 metadata:
   name: goldpinger
   namespace: default
+  annotations:
+    prometheus.io/scrape: 'true'
+    prometheus.io/port: '8080'
```

Comment:
Just one note - if you add the annotations on the service, then only one pod will be randomly scraped each time. I would recommend to put it directly on the pods, somewhere around the line 138.

Reply:
But metrics exposed at url/metrics which is not pod specific right? it's for everything

Reply:
what I mean we don't care about fetch from all, we care about fetch from anyone at any point of time

Reply:
No, each pod serves the metrics from its POV. When producing the graph on the UI, we ask one pod to ask all other pods for their version of events, and then aggregate it for us in a single image. For metrics, on the other hand, each pod is advertising what it's seen and then it's up to your grafana kung-fu to aggregate, slice and dice and process it.

Reply:
annotations moved from service to pod and tested